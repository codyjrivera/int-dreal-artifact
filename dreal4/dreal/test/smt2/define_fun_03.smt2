(set-logic QF_NRA)
(set-option :produce-models true)
(declare-const r1 Bool)
(declare-const r2 Bool)
(declare-const r3 Bool)
(declare-const r4 Bool)
(define-fun my_and ((x Bool) (y Bool)) Bool (and x y))
(assert (= (my_and true true) r1))
(assert (= (my_and true false) r2))
(assert (= (my_and false true) r3))
(assert (= (my_and false false) r4))
(check-sat)
(get-model)
