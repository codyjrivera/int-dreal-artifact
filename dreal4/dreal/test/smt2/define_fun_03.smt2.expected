delta-sat with delta = 0.001
r1 : True
r2 : False
r3 : False
r4 : False
(model
  (define-fun r1 () Bool true)
  (define-fun r2 () Bool false)
  (define-fun r3 () Bool false)
  (define-fun r4 () Bool false)
)
